pkg/imageset/updateinstaller.go

```diff
@@ -222,7 +222,10 @@ func (o *UpdateInstallerImageOptions) updateInstallerImage(installerImage, cliIm
 		logger.WithError(err).Error("failed to get ClusterDeployment")
 		return err
 	}
-	cd.Status.InstallerImage = &installerImage
+	// Set installer image only if it is not overrided by ImageSet or CD
```

Comment:
I am concerned that this check, as is, will preserve an out-of-date installer image even when the imageset job yields a different installer image.

Reply:
If we dont have this check, it will always override installer image to one from CLI. So we good to remove this feature at all, as it would always ignore values from ImageSet. 

The installer image is good only for cluster install. So its SRE responsibility to remove it from  from ImageSet when it is not needed anymore, and any new clusters will fallback into default logic. If we would use it again and again, I would be worried about this.

Reply:
What if we had the update-installer-image command take a parameter for the installer image override. If that parameter is set, then the commands writes that installer image to the clusterdeployment status instead of writing the installer image extracted from the release image. Bonus points if the extract-image script skips extracting the installer image when there is an override.

Reply:
sgtm